include/sp_vm_api.h

```diff
@@ -961,6 +1047,18 @@ namespace SourcePawn
      * as the cause.  
      */
     virtual cell_t BlamePluginError(IPluginFunction *pf, const char *msg, ...) = 0;
+   	
+	/**
+     * @brief Returns an IFrameIterator for the current call stack. Must
+	 * be freed by FreeFrameIterator()
+     */
+    virtual IFrameIterator *GetFrameIterator() = 0;

```

Comment:
Spacing in the comment looks corrupt - should be spaces instead of tabs. Let's call this "CreateFrameIterator" and add a comment like: "It is illegal to re-enter the VM while a frame iterator is active. The iterator must be processed and destroyed before continuing to run SourcePawn plugins."